### PR TITLE
ENH: Update CI for ITK v5.2rc03+

### PR DESCRIPTION
**Pull request by Leengit**
ENH: Update CI for ITK v5.2rc03+, including    
* ENH: Allow itkwidgets to be current version instead of 0.30.1 locked
* ENH: Add Python 3.9; Remove Python 3.5
* ENH: Specify CMake version 3.18.3; Xcode version 11.7.
* Build against ITK commit 7548a390d71dc25b80e792703ee3cfea17853a99.

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.0"
+            itk-git-tag: "v5.2rc03"
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml]: This is RC3, not RC3+.

Comment (Leengit) [reply .github/workflows/build-test-package.yml]: I just changed the pull request message and will change the commit message.  If there is elsewhere too, please let me know.

Comment (dzenanz) [reply .github/workflows/build-test-package.yml]: RC3 fails to compile on Windows, due to `lrintf` error.

Comment (dzenanz) [reply .github/workflows/build-test-package.yml]: I recomment to pick the commit from here:
https://github.com/InsightSoftwareConsortium/ITKMontage/commit/286934574028e2baaf997eacaa7e346a4693b492, i.e. `7548a390d71dc25b80e792703ee3cfea17853a99`.

Comment (Leengit) [reply .github/workflows/build-test-package.yml]: Thank you.  I am trying that now.

File: .github/workflows/build-test-package.yml [hunk 2]
```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.0"
+            itk-git-tag: "7548a390d71dc25b80e792703ee3cfea17853a99"
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:16]: Good, but now it is `RC3+`. I thought that would remind you of this commit 😄

Comment (Leengit) [reply .github/workflows/build-test-package.yml:16]: Thank you.  Fixed.

**Comment by Leengit** [comment]
This is currently all green on the [CDash](https://open.cdash.org/index.php?project=Insight&begin=2021-03-28&end=2022-03-28&filtercount=1&showfilters=1&field1=revision&compare1=63&value1=5672eb86b674ed65c5ca54e0740b7820e23fb071)board.  However, the build-windows-python-packages is failing in the checks for this pull request.  That appears to be because it is looking for Python 3.5, even though this project is looking for only 3.6-3.9.

**Comment by Leengit** [comment]
Is there some setting that I need to change at https://github.com/InsightSoftwareConsortium/ITKColorNormalization/ to change from Python 3.5?

**Review by dzenanz — APPROVED** [review]

**Pull request merged**